src/PluginTesting.ts

```diff
@@ -142,7 +142,7 @@ export async function testKeyStore(
     console.log(
       "expected metadata: ",
       keyMetadata.publicKey,
-      "and test stored shit got back: ",
+      "but received: ",
       testMetadata[0],
     );
```

Comment:
I don't think this console.log was meant to be checked in, so we can get rid of it entirely. Nice find!

Reply:
Great! Just pushed a new commit.